core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -126,33 +126,38 @@ fun LpPostCard(
     onFavoriteClick: () -> Unit
 ) {
     Card(
-        modifier = modifier.padding(15.dp),
+        modifier = modifier.padding(LeasePertTheme.sizes.smallSize),
         colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primaryContainer),
-        border = BorderStroke(1.dp, MaterialTheme.colorScheme.onPrimary),
-        elevation = CardDefaults.cardElevation(1.dp),
+        border = BorderStroke(LeasePertTheme.sizes.stroke, MaterialTheme.colorScheme.onPrimary),
+        elevation = CardDefaults.cardElevation(LeasePertTheme.sizes.stroke),
     ) {
         Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(15.dp, 15.dp, 15.dp, 5.dp)
+                .padding(
+                    LeasePertTheme.sizes.smallSize,
```

Comment:
please add the name of each property like: `start =  LeasePertTheme.sizes.smallSize`

Reply:
Working on the comments

Reply:
Comment resolved